### ISSUE TITLE
Looking for a new maintainer

### ISSUE THREAD

**Issue by raiguard** [raiguard/ModernGadgets#288]
I think it's time. I have not used windows in over a year, and this project is languishing as a result. I have no motivation to work on it any more. I think it would be best to pass the torch to another so they can keep ModernGadgets alive.

I'm super grateful to everyone who loves and uses ModernGadgets. Thank you for your support! Working on this project helped me a lot through my teenage years, and kickstarted my interest in software development as a whole.

If you're interested in maintaining the project, respond here and we'll work out the details.

**Comment by raiguard** [comment]
Some more context for this decision:
- I use Linux now, making it difficult to test ModernGadgets since I have to do so in a VM
- I am starting university soon, and need to decrease my cognitive load on personal projects as much as I can
- I am migrating from GitHub to Sourcehut, but don't want to uproot the community that has grown here
- I don't want this project to die. Thousands of people use ModernGadgets every day, and I want them to continue to be able to do so.

It feels like I'm putting my child up for adoption, but it's the best thing to do.

**Comment by SilverAzide** [comment]
Hm... 🤔 This would be quite a commitment for whoever is willing to take it on. This is a very large suite to manage and maintain! The Rainmeter community will miss your contributions and insights...

**Comment by SilverAzide** [comment]
I'd volunteer to take this on, but due to the size and popularity of ModernGadgets I'm afraid it would eat my life.  It's hard enough maintaining the stuff I have now.

You might want to post this on your Rainmeter [ModernGadgets thread](https://forum.rainmeter.net/viewtopic.php?t=24381) to get more eyeballs on it.  That's still a pretty active thread.

**Comment by raiguard** [comment]
Good idea.

Yeah, I don't want to throw it on you, you already have your hands full. ModernGadgets is pretty similar feature-wise to your suite, the main differences are in presentation. Adding a flat theme to your gadgets would probably be enough to bring over most users.

**Comment by SilverAzide** [comment]
Yes, I was also thinking that theming might help!  I can break out my old Gadgets "Modernizer" patch to flatten things, LOL.  But more seriously, I could try using your main font and flattening/de-borderizing to get part-way there.  I should have bitten the bullet long ago like you did and used style sheets.

**Comment by raiguard** [comment]
The more I think about it, the less I like the idea of handing the repository to someone I don't trust. You (SilverAzide) would be the only one I would fully trust to do it, but we already decided against that. In light of this, I'm going to officially close this repository, and encourage everyone to use your suite instead.

If you want, I can help you make a flat theme for your gadgets, though you know _far_ more about Rainmeter than I do, so you probably won't need my help. Thank you for everything you've done over the years!

If anyone would like to contact me, you can reach me at ch@raiguard.me. My inbox is open.
